vmdb/app/models/miq_request_workflow.rb

```diff
@@ -949,7 +949,7 @@ def each_ems_metadata(ems_ci = nil, klass = nil, &_blk)
 
   def get_ems_metadata_tree(src)
     @ems_metadata_tree ||= begin
-      st = Time.now
+      st = Time.zone.now
```

Comment:
Is this change needed?  Since we are just using it to calculate the timing I do not see why zone would matter.